nilearn_sandbox/reporting/papaya.py

```diff
@@ -0,0 +1,45 @@
+import base64
+import nibabel
+import numpy as np
+import os
+from nilearn._utils import check_niimg
+
+
+def _get_64(niimg):
+    f = check_niimg(niimg)
+    data = f.get_data().astype(float)
+    data = data + data.min()
+    data = data / data.max() * 254.
+    data = data.astype(np.uint8)
+    f = nibabel.Nifti1Image(data, f.get_affine())
+    f.to_filename('_maps.nii.gz')
+    with open('_maps.nii.gz', 'br') as f:
+        b64 = base64.b64encode(f.read())
+    os.unlink('_maps.nii.gz')
+    return b64.decode('utf-8')
+
+
+def papaya_viewer(html_path, maps_niimg):
```

Comment:
This seems to create the HTML files, but doesn't launch any viewer (if I understand properly).

If that's right, then a name like `write_papaya_html` seems better. I expected the browser to be launched here.